sources/packages/api/src/cra-integration/cra-personal-verification.service.ts

```diff
@@ -203,6 +221,7 @@ export class CRAPersonalVerificationService {
       surname: student.user.lastName,
       givenName: student.user.firstName,
       birthDate: student.birthDate,
+      userId: student.user.id,
```

Comment:
I believe that we should remove the userId from the model. Please see the suggested method to update the sent records.